extensions/wikia/AdEngine/js/provider/monetizationService.js

```diff
@@ -28,26 +30,42 @@ define('ext.wikia.adEngine.provider.monetizationService', [
 		return false;
 	}
 
-	function fillInSlot(slot, slotElement, success, hop) {
-		log(['fillInSlot', slot, slotElement], 'debug', logGroup);
+	function initialize(success) {
+		log(['initialize', 'send request'], 'debug', logGroup);
+		nirvana.getJson(
+			'MonetizationModule',
+			'getModules',
+			{
+				adEngine: true,
+				articleId: window.wgArticleId, // TODO: use ad context
+				fromSearch: window.fromsearch, // TODO: convert window.fromsearch to a module
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1135)